app/models/energy/construction.py

```diff
@@ -338,79 +394,79 @@ class EnergyWindowMaterialBlind(BaseModel):
         ' to be 100% diffuse (i.e., slats are translucent). The default value is 0.'
     )
 
-    front_beam_solar_reflectance: float = Schema(
+    front_beam_solar_reflectance: float=Schema(
```

Comment:
We should follow the convention here of not using the word `front` in the name (only for `back`)